doc/classes/AnimationNodeAdd2.xml

```diff
@@ -4,7 +4,9 @@
 		Blends two animations additively inside of an [AnimationNodeBlendTree].
 	</brief_description>
 	<description>
-		A resource to add to an [AnimationNodeBlendTree]. Blends two animations additively based on an amount value in the [code][0.0, 1.0][/code] range.
+		A resource to add to an [AnimationNodeBlendTree]. Blends two animations additively based on an amount value.
```

Comment:
```suggestion
		A resource to add to an [AnimationNodeBlendTree]. Blends two animations additively based on a numerical value.
```

Reply:
I changed it to `based on the amount value.`